cpu/sam0_common/periph/flashpage.c

```diff
@@ -45,6 +42,38 @@ void flashpage_write(int page, void *data)
         PAC1->WPCLR.reg = NVMCTRL_PAC_BIT;
     }
 #endif
+}
+
+void flashpage_write_raw(void *target_addr, void *data, size_t len)
+{
+    /* The actual minimal block size for writing is 16B, thus we
```

Comment:
actually, we assert that we write in multiples of that.

Reply:
Changed.